cmd/api-response-multipart.go

```diff
@@ -20,6 +20,21 @@ package cmd
 
 import "net/http"
 
+// Represents additional fields necessary for ErrPartTooSmall S3 error.
+type completeMultipartAPIError struct {
```

Comment:
Do not introduce these custom errors just yet, i am fixing this differently. For now do not validate other fields other than the common ones that occur in `APIErrorResponse`